gpu_cmode.lua

```diff
@@ -130,7 +130,12 @@ function slurm_spank_task_init_privileged(spank)
                    cmode, device_ids)
     local cmd = nvs_path .. " -c " .. cmodes_index[cmode] ..
                             " -i " .. device_ids
-    local ret = tonumber(os.execute(cmd))
+    local ret = 0
+    if _VERSION == 'Lua 5.1' then
```

Comment:
Maybe the test should be for Lua versions <= 5.1, instead of just 5.1?
AFAICT, earlier versions of Lua all returned a single value.

Reply:
I have no clue why this works but can do `_VERSION <= "Lua 5.1"` . I would think because it's a string it wouldn't work but it does.  I made this change

Lua 5.1:

```
$ lua
Lua 5.1.4  Copyright (C) 1994-2008 Lua.org, PUC-Rio
> if _VERSION <= "Lua 5.1" then
>> print("YES")
>> else
>> print("NO")
>> end
YES
```

Lua 5.3:

```
$ lua
Lua 5.3.4  Copyright (C) 1994-2017 Lua.org, PUC-Rio
> if _VERSION <= "Lua 5.1" then
>> print("YES")
>> else
>> print("NO")
>> end
NO
```

Reply:
Ah, that's awesome! :grin: